<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>440</width>
    <height>494</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>LVDT</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>470</y>
     <width>221</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>waiting for the package</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>560</y>
     <width>80</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>mode 0</string>
   </property>
  </widget>
  <widget class="QPushButton" name="openMyComBtn">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>10</y>
     <width>80</width>
     <height>20</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="text">
    <string>打开串口</string>
   </property>
  </widget>
  <widget class="QPushButton" name="closeMyComBtn">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>10</y>
     <width>80</width>
     <height>20</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="text">
    <string>关闭串口</string>
   </property>
  </widget>
  <widget class="QPushButton" name="sendMsgBtn">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>50</y>
     <width>80</width>
     <height>20</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="text">
    <string>设置从机IP</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="sendMsgLineEdit">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>50</y>
     <width>201</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="portNameComboBox">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>10</y>
     <width>101</width>
     <height>22</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="currentText">
    <string/>
   </property>
   <property name="currentIndex">
    <number>-1</number>
   </property>
  </widget>
  <widget class="QLabel" name="comNum">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>54</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>串口号：</string>
   </property>
  </widget>
  <widget class="QLabel" name="iPAddr">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>50</y>
     <width>71</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>从机IP地址：</string>
   </property>
  </widget>
  <widget class="QTextBrowser" name="textBrowser">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>170</y>
     <width>201</width>
     <height>321</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="mode2Button">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>550</y>
     <width>80</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>mode 1</string>
   </property>
  </widget>
  <widget class="QTextBrowser" name="textBrowserCom">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>170</y>
     <width>201</width>
     <height>321</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="filterTypeComboBox">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>90</y>
     <width>81</width>
     <height>21</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="currentText">
    <string>None</string>
   </property>
   <item>
    <property name="text">
     <string>None</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>FIR</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>IIR</string>
    </property>
   </item>
  </widget>
  <widget class="QLineEdit" name="lineEdit">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>530</y>
     <width>81</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QCheckBox" name="ehCheckBox">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>140</y>
     <width>73</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Adobe Caslon Pro</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>以太网</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="comCheckBox">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>140</y>
     <width>63</width>
     <height>25</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Adobe Caslon Pro</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>串口</string>
   </property>
  </widget>
  <widget class="QComboBox" name="filterOrderCombox">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>530</y>
     <width>51</width>
     <height>22</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="editable">
    <bool>false</bool>
   </property>
   <property name="currentText">
    <string>10</string>
   </property>
   <item>
    <property name="text">
     <string>10</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>20</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>40</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>60</string>
    </property>
   </item>
  </widget>
  <widget class="QComboBox" name="wcComboBox">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>90</y>
     <width>61</width>
     <height>22</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <item>
    <property name="text">
     <string>250Hz</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>500Hz</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>1KHz</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>2KHz</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>90</y>
     <width>61</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>截止频率：</string>
   </property>
  </widget>
  <widget class="QComboBox" name="sendFrqCombox">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>120</y>
     <width>81</width>
     <height>22</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <item>
    <property name="text">
     <string>1KHz</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>500Hz</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>250Hz</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>120</y>
     <width>71</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>发送频率：</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>91</y>
     <width>72</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>数字滤波器：</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>510</y>
     <width>31</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>阶次：</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_2">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>120</y>
     <width>61</width>
     <height>20</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="text">
    <string>保存数据</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_3">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>120</y>
     <width>61</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>开始</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_4">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>120</y>
     <width>71</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>停止</string>
   </property>
  </widget>
  <widget class="QPushButton" name="sethostIP">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>540</y>
     <width>80</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>设置主机IP</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="sethostIPlineEdit">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>530</y>
     <width>201</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="iPAddr_2">
   <property name="geometry">
    <rect>
     <x>520</x>
     <y>540</y>
     <width>71</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>主机IP地址：</string>
   </property>
  </widget>
  <zorder>label</zorder>
  <zorder>pushButton</zorder>
  <zorder>openMyComBtn</zorder>
  <zorder>closeMyComBtn</zorder>
  <zorder>sendMsgBtn</zorder>
  <zorder>sendMsgLineEdit</zorder>
  <zorder>portNameComboBox</zorder>
  <zorder>comNum</zorder>
  <zorder>iPAddr</zorder>
  <zorder>textBrowser</zorder>
  <zorder>mode2Button</zorder>
  <zorder>textBrowserCom</zorder>
  <zorder>filterTypeComboBox</zorder>
  <zorder>lineEdit</zorder>
  <zorder>ehCheckBox</zorder>
  <zorder>comCheckBox</zorder>
  <zorder>filterOrderCombox</zorder>
  <zorder>wcComboBox</zorder>
  <zorder>label_4</zorder>
  <zorder>label_3</zorder>
  <zorder>label_2</zorder>
  <zorder>sendFrqCombox</zorder>
  <zorder>label_5</zorder>
  <zorder>pushButton_2</zorder>
  <zorder>pushButton_3</zorder>
  <zorder>pushButton_4</zorder>
  <zorder>sethostIP</zorder>
  <zorder>sethostIPlineEdit</zorder>
  <zorder>iPAddr_2</zorder>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
